examples/native/components/ControlExamples/Boolean/Boolean.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
Looks like this & the other controls stories aren't picked up in the examples. Should we change `examples/native/storybook.requires.js` to contain

```
const getStories = () => {
  return [
    require('./components/PromiseTest/Button.stories.tsx'),
    require('./other_components/AnotherButton/AnotherButton.stories.tsx'),
    require('./components/ControlExamples/Boolean/Boolean.stories.tsx'),
    require('./components/ControlExamples/Color/Color.stories.tsx'),
    require('./components/ControlExamples/Number/Number.stories.tsx'),
    require('./components/ControlExamples/Object/Object.stories.tsx'),
    require('./components/ControlExamples/Select/Select.stories.tsx'),
    require('./components/ControlExamples/Text/Text.stories.tsx'),
  ];
};
```

or is there a better way to handle it?

Reply:
Well the requires file is auto generated, it should work if the regex is correct in the main.js.

Reply:
Good point! One of these days I might want to start reading comments 😅 :

```
/*
  do not change this file, it is auto generated by storybook. 
*/
```

The regexes are

```
  stories: [
    './components/*/*.stories.?(ts|tsx|js|jsx)',
    './other_components/AnotherButton/AnotherButton.stories.tsx',
  ],
```

So we could change the upper one to `'./components/**/*.stories.?(ts|tsx|js|jsx)',` to account for the deeper directory structure in `ControlExamples` - or group the control stories under `Controls` and have a flatter directory structure, whichever we want.